src/posit/connect/content.py

```diff
@@ -189,10 +189,18 @@ def dashboard_url(self) -> str:
     def app_role(self) -> str:
         return self.get("app_role")  # type: ignore
 
+    @property
+    def owner(self) -> dict:
+        return self.get("owner", {})
+
     @property
     def permissions(self) -> Permissions:
         return Permissions(self.config, self.session, content_guid=self.guid)
 
+    @property
+    def tags(self) -> List[dict]:
```

Comment:
Followup issue for this property: https://github.com/posit-dev/posit-sdk-py/issues/164